torchquad/integration/vegas.py

```diff
@@ -42,6 +42,8 @@ def integrate(
     ):
         """Integrates the passed function on the passed domain using VEGAS.
 
+        If the integrand output is far away from zero, i.e. lies within [a, a+c] for an absolutely big constant a and small constant c, VEGAS does not adapt well to the integrand. Shifting the integrand so that it is close to zero may improve the accuracy of the calculated integral in this case.
```

Comment:
```suggestion
        If the integrand output is far away from zero, i.e. lies within [a, a+c] for an constant a with large absolute value and small constant c, VEGAS does not adapt well to the integrand. Shifting the integrand so that it is close to zero may improve the accuracy of the calculated integral in this case.
```
Sounds too much like some former presidents otherwise :)

Reply:
done.